include/avif/avif.h

```diff
@@ -846,6 +947,12 @@ typedef struct avifIOStats
     size_t colorOBUSize;
     // Size in bytes of the AV1 image item or track data containing alpha samples.
     size_t alphaOBUSize;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    // Size in bytes of the AV1 image item containing gain map samples.
+    size_t gainMapOBUSize;
+#else
+    size_t reserved;
+#endif
```

Comment:
IMPORTANT: If it is necessary, or very convenient, to store the gain map size in the `avifIOStats` struct, we need to prepare for this in the libavif 1.0.0 release.

There are two options:

(1) Add enough reserved fields at the end of the `avifIOStats` struct.

(2) Change the declaration of the `ioStats` member of `avifDecoder` and `avifEncoder` from

```
    avifIOStats ioStats;
```

to

```
    avifIOStats * ioStats;
```

and let `avifDecoderCreate()` and `avifEncoderCreate()` allocate the `avifIOStats` struct.

Alternatively, we can store the gain map size in the `avifDecoderData` and `avifEncoderData` structs, and add getter functions to extract the gain map size from `avifDecoder` or `avifEncoder`.

Reply:
I removed this field. I don't have any particular need for it, I simply added it to mirror the other two fields.
I was chatting with Yannis and he was wondering if we could get rid of this struct entirely.